pkg/terminal/command.go

```diff
@@ -141,6 +142,9 @@ Called with more arguments it will execute a command on the specified goroutine.
 	[goroutine <n>] [frame <m>] print <expression>
 
 See $GOPATH/src/github.com/derekparker/delve/Documentation/cli/expr.md for a description of supported expressions.`},
+		{aliases: []string{"whatis"}, allowedPrefixes: scopePrefix, cmdFn: whatisCommand, helpMsg: `Prints type of an expression.
+		
+		whatis <expression>.`},
```

Comment:
Technically this won't eval an entire expression, let's just put `<var name>` or something like that instead.

Reply:
Ah, I see that now, seemed when I looked (and tested) before other args were getting ignored. Either way, this seems fine.